jetstream/config.py

```diff
@@ -406,8 +467,14 @@ def resolve(
                 or [mozanalysis.experiment.AnalysisBasis.ENROLLMENTS],
             )
         else:
+            params = {
+                param: spec.parameters.definitions[param].value
+                or spec.parameters.definitions[param].default
+                for param in spec.parameters.definitions
+            }
+
```

Comment:
Ah I see what's going on here. There is also a `params` variable introduced on line 493:

```
for statistic_name, params in self.statistics.items():
```

Maybe rename this one?